app/templates/reset_password.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html" %}
+{% block content %}
+    <div class="container">
+        <h5>Please choose a new password:</h4>
+        {% if message %}
+            <div class="alert" style="color: red;">{{ message }}</div>
+        {% endif %}
+        <form method="post">
+            <div class="row">
+                <label for="username" class="col-sm-2 col-form-label">Username</label>
+                <div class="col-auto">
+                    <input type="text" class="form-control" name="username" id="username"  placeholder="Required" autofocus><br>
+                </div>
+            </div>
+            <div class="row">
+                <label for="emil" class="col-sm-2 col-form-label">password</label>
+                <div class="col-auto">
+                    <input type="text" class="form-control" name="password" id="password"  placeholder="Required"><br>
+                </div>
+                <div class="col-auto">
+                    <span id="PasswordHelpInline" class="form-text">
+                        Must be between 3 to 20 characters.
+                    </span>
+                </div>
+            </div>
+            <div class="row">
+                <label for="confirm_password" class="col-sm-2 col-form-label">Confirm Password</label>
+                <div class="col-auto">
+                    <input type="text" class="form-control" name="confirm_password" id="confirm_password" placeholder="Required"><br>
```

Comment:
multi-words id names should like classes be separated with a dash and not an underscore. So "confirm-password".

Reply:
This Id is being sent directly to pydantic class from the route. Changing that means adding few totally unnecessary code lines in the route, just to change that comma back to a syntax that pydantic class can work with. I prefer this solution